test/test.js

```diff
@@ -0,0 +1,29 @@
+import Person from '../src/Person.js';
+import {Student} from '../src/Student.js';
+import {Teacher} from '../src/Teacher.js';
+import { expect } from 'chai';
+
+describe('Person class: create a person',() => {
+    
+    let gbenga = new Student('gbenga','ode','30/06/1993','male','Bright Minds High');
+
+    it('The person should be a type of "object", and an instance of the "person" class',() => {
+        expect(typeof gbenga).to.eql('object');
+    });
+
+    it('should return "24"',() => {
+        expect(gbenga.age).to.eql(24);
+    });
+
+    it('should return "male"', () => {
+        expect(gbenga.gender).to.eql('male');
+    });
+
+    it('should return "Hi!, I am ode gbenga and I am a student of Bright Minds High"', () => {
+        expect(gbenga.introduction()).to.eql("Hi!, I am ode gbenga and I am a student of Bright Minds High");
+    });
+
+    it('should return "ode gbenga"',() => {
+        expect(gbenga.fullName).to.eql('ode gbenga');
+    });
+});
```

Comment:
I can see only five test cases. You can do better than that.

Reply:
Why are using a symbol to represent your variables?

Reply:
Yes I can definitely do better .I'll write more test cases. I had a slight set back and had to start over.  Five was what I could come up with within the time I had left.

Reply:
I'm using symbols to help encapsulate the properties

Reply:
I'll also take note of the branch naming convention